rules/Transform/Rector/Class_/TestWithToDataProviderRector.php

```diff
@@ -0,0 +1,209 @@
+<?php
```

Comment:
Please also check if the files are placed in the correct folders path :)

Note: in case it's, then I need to provide other PR moving this file to rules:
https://github.com/rectorphp/rector-phpunit/blob/main/src/Transform/AssertEqualsOrAssertSameFloatParameterToSpecificMethodsTypeRector.php, when I was doing this feature, I realised that is in the wrong place :/

Reply:
Ideally put it here, so we use existing set and avoid too detailed granulation.

https://github.com/rectorphp/rector-phpunit/tree/main/rules/CodeQuality/Rector

Once you move it, it's ready to merge

Reply:
Do you mean this new rule ([TestWithToDataProviderRector.php](https://github.com/rectorphp/rector-phpunit/pull/283/files#diff-6d31e06ec7631468c3201fef89d14a831a0c792c782389beff876353cba3ad94)) or this other one that is already merged ([AssertEqualsOrAssertSameFloatParameterToSpecificMethodsTypeRector.php](https://github.com/rectorphp/rector-phpunit/blob/main/src/Transform/AssertEqualsOrAssertSameFloatParameterToSpecificMethodsTypeRector.php))?

or do you mean both of them?

Reply:
This one in this PR - `TestWithToDataProviderRector`

Reply:
Hello @TomasVotruba Dobré ráno :)

how are you, I hope you have had a great start of week :)

I just moved the code to CodeQuality and I also added it in `config/sets/phpunit-code-quality.php`, I hope everything as expected 😄 

Thanks